templates/__common__/config/tasks/graphics-meta.js

```diff
@@ -159,6 +176,15 @@ const parseGraphic = async (
   const tags = await (await getText({ key: 'tags', page })).split(',');
   let credits = await getText({ key: 'credit', page });
 
+  // determine type of metadata, we get metadata for features and embeddable graphics
+  let type;
+  if (await checkForAttribute({ key: 'feature', page })) {
+    type = 'feature';
+  }
+  if (await checkForAttribute({ key: 'graphic', page })) {
+    type = 'graphic';
+  }
+
```

Comment:
if we wanted assume it's either graphic or feature since it got to this point, we could streamline this a bit maybe:
```js
// assume the type is graphic unless data-feature is present
const type = (await page.$('[data-feature]')) ? 'feature' : 'graphic';
  ```